docs/components/connectors/out-of-the-box-connectors/graphql.md

```diff
@@ -0,0 +1,212 @@
+---
+id: graphql
+title: GraphQL Connector
+description: Execute a GraphQL query or mutation from your BPMN process.
+---
+
+The **GraphQL Connector** allows you to execute a GraphQL query or mutation from your BPMN process.
+
+## Prerequisites
+
+To use the GraphQL Connector, a GraphQL server must be configured. A respective authentication method needs to be configured. Follow [these steps in the appendix](#appendix) to learn how to set this up.
```

Comment:
There is no appendix?

What do you mean by this?

I would just state something like:

```
The GraphQL Connector allows you to connect to a GraphQL API endpoint. In order to use the GraphQL connector, you need to know the GraphQL endpoint URL, authentication, and avaiable API methods.
```